domain.tf

```diff
@@ -6,7 +6,7 @@ and AWS Route 53
 
 resource "tls_private_key" "lets_encrypt_private_key" {
   algorithm = "RSA"
-  rsa_bits = "4096"
+  rsa_bits  = "4096"
```

Comment:
I think this is the bit with the merge conflict. we were having issues specifying the key length, so when you resolve the conflicts, please omit the key length in the resource and the rest in this file.